lib/reek/cli/command.rb

```diff
@@ -1,16 +0,0 @@
-module Reek
-  module CLI
-    #
-    # Base class for all commands
-    #
-    class Command
```

Comment:
I'm not sure I follow how using a mixin resolves a), since you're jumping between module and class?

Does b) mean the Command pattern violates Liskov?